data.sql

```diff
@@ -8,3 +8,16 @@ INSERT INTO animals(id, name, date_of_birth, escape_attempts,neutered,weight_kg)
 
 INSERT INTO animals(id, name, date_of_birth, escape_attempts,neutered,weight_kg) VALUES(4, 'Devimon', '05-12-2017', 5, TRUE, 11);
 
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(5,'Charmander','02-08-2020',0,FALSE,11.0);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(6,'Plantmon','11-15-2021',2,TRUE,5.7);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(7,'Squirtle','04-02-1993',3,TRUE,12.13);
+
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(8,'Angemon','06-12-2005',1,FALSE,45);
```

Comment:
- [OPTIONAL] According to the project requirements, these values for weight are negative values, please kindly fix them.